client/src/components/RatingsAndReviews/ReviewsList/ReviewsListItem.scss

```diff
@@ -0,0 +1,12 @@
+.review_list {
+  padding: 10px 0;
+  border-bottom: 1px solid #888;
+}
+
+.reviewer_name {
+  font-size: 12px;
+}
+
+.reviewer_helpful {
+  font-size: 12px;
+}
```

Comment:
Could you please add a new line at the end of the file? Just a tiny best practice thing.

Reply:
done!